pkg/clusternetwork/config.go

```diff
@@ -16,6 +17,7 @@ const ovnKubernetesNetwork = "OVNKubernetes"
 type ClusterNetworkConfig interface {
```

Comment:
Side comment: I wish we had just called this `Config`. `clusternetwork.ClusterNetworkConfig` stutters :-(.

Reply:
Next PR for sure